event_types/event_types.py

```diff
@@ -15,12 +15,29 @@
 from joblib import dump, load
```

Comment:
In line 1467 you refer to the "this_e_range" variable, but it doesn't exist there. Maybe it is a dead function? Perhaps remove if the code is obsolete (or fix of course)

Reply:
This is a dead function, yes. It is only useful for tree based models and I don't use them anymore. I can remove the function, no problem. If we need it again we can go back to the history and retrieve it.